plugins/egis-moc/README.md

```diff
@@ -0,0 +1 @@
+# fixme
```

Comment:
@jasonhouang this needs filling out, you can use one of the other USB files as a template -- e.g. `hughski-colorhug/README.md`